crossengage/client.py

```diff
@@ -88,7 +128,7 @@ def update_users_bulk(self, users):
         :return: json dict response
         """
         payload = {'updated': users}
-        self.request_url = self.API_URL + self.USER_ENDPOINT + 'batch'
+        self.request_url = "{0}/{1}".format(self.API_URL, self.USER_BULK_ENDPOINT)
```

Comment:
I would put `version=` attribute here for better understanding which api version I am using

Reply:
You mean `return self.__create_request(payload=payload, request_type=self.REQUEST_POST, version="v1")`?

Reply:
yes, just for better viewing which method calls which api version